src/pages/home/report/ReportActionsView.js

```diff
@@ -102,28 +100,28 @@ class ReportActionsView extends React.Component {
 
         // This callback is triggered when a new action arrives via Pusher and the event is emitted from Report.js. This allows us to maintain
         // a single source of truth for the "new action" event instead of trying to derive that a new action has appeared from looking at props.
-        this.unsubscribeFromNewActionEvent = Report.subscribeToNewActionEvent(this.props.report.reportID, (isFromCurrentUser, currentLastSequenceNumber) => {
+        this.unsubscribeFromNewActionEvent = Report.subscribeToNewActionEvent(this.props.report.reportID, (isFromCurrentUser, currentLastReportActionID) => {
```

Comment:
I'm not exactly sure how to hit each one of these conditions in order to test them. @rushatgabhane do you know?

Reply:
Or @roryabraham